.github/ISSUE_TEMPLATE/change-request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F680 Change Request"
+about: Any suggestion? Want to add something new?
+---
+
+## Change Request
+
+**This repository is ONLY used to solve issues related to DOCS.
+For other issues (related to TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**
```

Comment:
```suggestion
For other issues (related to TiDB), please move to [other repositories](https://github.com/pingcap/).**
```

Reply:
All TiKV related repositories are now in a separate place [here](https://github.com/tikv).